docs/source/examples/checks/distribution/trust_score_comparison.ipynb

```diff
@@ -2,106 +2,136 @@
  "cells": [
```

Comment:
<p>verity -&gt; variety</p><p> </p><ol><li>"Detecting "problematic samples" before labels become available - prediction with low probability are assumed to be more likely to fail.</li></ol><p>would change the end "with low probability are more likely to be wrong"</p><p> </p><p>"This is because the methods and loss functions used by these models are often not designed to produce actual probabilities, and model performance is measured by metric that mostly rely only on predicted class. " -&gt; last part unclear</p><p> </p><p>--</p><p>link to paper - is it external?</p><p>--</p><p>last point (2):</p><ol><li>Identifying a degradation between the trust score on the test data when comparing it to the training data, which may indicate that the model will perform worse on test compared to train and serves as a method to detect concept drift. This condition is useful especially for cases when the test labels are not available, such as when performing inference on new and unknown data.</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/644/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>yes links are external </p>